### ISSUE TITLE
[BUG] All nondecorative images need alternative text

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5180]
**Describe the bug**
When using the screen reader to navigate through members of a group it gets really redundant quickly to hear "User image, double tap to zoom. Double tap to activate." This is not providing any valuable information to the Screen reader user. 

**To Reproduce**
1. Go to Groups
2. Click on Members
3. Swipe through with TalkBack or VoiceOver

**Expected behavior**
Useful alternative text is read. 

**Actual behavior** 
When using the screen reader to navigate through members of a group it gets really redundant quickly to hear "User image, double tap to zoom. Double tap to activate." This is not providing any valuable information to the Screen reader user. 

Additionally, on many main Group pages of public groups there are Admins listed at bottom. For these, the screen reader reads "unlabeled, double tap to activate." Then when the user double taps they hear a click noise and nothing else happens. 

**Screenshots**

**Device**
•	OS: Android Version 15
•	Device: Google Pixel 8a
•	Browser: Chrome 136.0.7103.87

**Additional context**

![Image](https://github.com/user-attachments/assets/328810fc-e0de-443e-aace-d027c22273e7)

Additionally, on many main Group pages of public groups there are Admins listed at bottom. For these, the screen reader reads "unlabeled, double tap to activate." Then when the user double taps they hear a click noise and nothing else happens. 

In the Web app you can NOT even gesture through.

**Comment by TodorBachvarov** [comment]
@acovher2 in version 7.3.22 I've fixed the Alt text input for the users profile photo. Now if the user have set an alt text it will be pronounced. In addition if there is no alt text set for the image the screen reader will pronounce "Profile image". If the Image is not set at all and the default placeholder is shown, the screen reader will skip this image as it is decorative.
